README.md

```diff
@@ -87,6 +87,7 @@ Clears content on window.
       <td><a href="#p-rm">rm</a></td>
       <td><a href="#q-tail">tail</a></td>
       <td><a href="#r-touch">touch</a></td>
+      <td><a href="#s-find">find</a></td>
```

Comment:
It should be after [this](https://github.com/Idnan/bash-guide/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R74). As i have sorted all the commands in alphabetical order.